config/packages/doctrine.yaml

```diff
@@ -13,6 +13,12 @@ doctrine:
                 password: "%database_middleware_password%"
                 server_version: "%database_server_version%"
                 charset: utf8
+                options:
+                    # Enabling the STRICT_ALL_TABLES SQL mode. To prevent 'magic' truncation problems where string
+                    # values like the identity name id would be truncated after 255 characters without notice. Enabling
+                    # STRICT_ALL_TABLES changes this behaviour, and raises SQLSTATE[22001] 'Bad response' when this
+                    # occurs.
+                    1002: 'SET sql_mode="STRICT_ALL_TABLES"'
```

Comment:
1. This will override any sql modes that were already set. Is that intentional? In de mariadb documentation they use CONCAT to add a flag. I.e.: `SET @@SQL_MODE = CONCAT(@@SQL_MODE, ',STRICT_ALL_TABLES');`
See: https://mariadb.com/kb/en/sql-mode/#examples

2. Please add a comment  that the magic constant `1002` is `PDO::MYSQL_ATTR_INIT_COMMAND`